components/ol/OlMap.tsx

```diff
@@ -0,0 +1,47 @@
+import "../../node_modules/ol/ol.css";
```

Comment:
would be better omit node_modules section when importing css files from node_modules